asreviewcontrib/covid19/datasets.py

```diff
@@ -9,46 +9,39 @@ class Cord19Dataset(BaseDataSet):
     topic = "Covid-19"
     license = "Covid dataset license"
     link = "https://pages.semanticscholar.org/coronavirus-research"
-    last_update = "2020-03-20"
+    last_update = "2020-03-27"
     description = "A free dataset on publications on the corona virus."
     img_url = ("https://pages.semanticscholar.org/hs-fs/hubfs/"
                "covid-image.png?width=300&name=covid-image.png")
     link = "https://pages.semanticscholar.org/coronavirus-research"
     year = 2020
 
 
-class Cord19DatasetV4(Cord19Dataset):
-    dataset_id = "cord19-v4"
-    title = "CORD-19 v4"
-    sha512 = ("517e2399767aa1d387baaa07c42ef6ac9a5aec1e3a41f832974ee712413"
-              "272429f2a5ea658b32bb7330becac70df1ee5262ae1ddebb258a02aaaa2"
-              "d4b47335cc")
-
-    date = "2020-03-20"
+class Cord19DatasetV5(Cord19Dataset):
+    dataset_id = "cord19-v5"
+    title = "CORD-19 v5"
+    date = "2020-03-27"
     statistics = {
-        "n_papers": 44220,
-        "n_missing_titles": 224,
-        "n_missing_abstracts": 8414,
+        "n_papers": 45774,
+        "n_missing_titles": 157,
+        "n_missing_abstracts": 7861,
     }
-    url = ("https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/2020-03-20/metadata.csv")
+    url = ("https://ai2-semanticscholar-cord-19.s3-us-west-2.amazonaws.com/2020-03-27/metadata.csv")  # noqa
+    sha512 = ("a6d8e7a4dd3f0194840129b7368fd97354b44a44fc0d06addfc9a27f3c08e19ef88ff568effb92994895353f7e9d771932f354bd619ee2242d4262f7561f41fe")  # noqa
 
 
-class Cord19DatasetV4_Dec2019(Cord19Dataset):
-    dataset_id = "cord19-v4-2020"
-    title = "CORD-19 v4 since Dec. 2019"
-    last_update = "2020-03-22"
+class Cord19DatasetV5_Dec2019(Cord19Dataset):
+    dataset_id = "cord19-v5-2020"
+    title = "CORD-19 v5 since Dec. 2019"
+    last_update = "2020-03-30"
```

Comment:
2020-03-27?

Reply:
Thanks - I entered the date the file was created (by subsetting from the larger file and adding a column), not the date the original file was released. 

If we want to stick to the date of update of the larger database itself, this can be changed to 2020-03-27.

Reply:
Thanks for your reply. I prefer 2020-03-27, but your remark is vulnerable.